packages/clay-navigation/stories/index.tsx

```diff
@@ -0,0 +1,125 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import '@clayui/css/lib/css/atlas.css';
+import ClayNavigation from '../src';
+import React from 'react';
+import {number} from '@storybook/addon-knobs';
+import {storiesOf} from '@storybook/react';
+
+const spritemap = require('@clayui/css/lib/images/icons/icons.svg');
+
+storiesOf('ClayNavigation', module)
```

Comment:
I would be curious to know how `Breadcrumbs` works when controlled with `onClick`, we may have an example of usage just like pagination.

Reply:
Sent!